src/main/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/exceptions/CreateLocationConflictException.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.pip.data.management.errorhandling.exceptions;
+
+public class CreateLocationConflictException extends RuntimeException {
```

Comment:
I'd add a java doc here to describe what this is used for

Reply:
Added java doc